src/boss/cloudvm/worker.go

```diff
@@ -339,7 +353,12 @@ func (w *Worker) runCmd(command string) {
 
 	tries := 10
 	for tries > 0 {
-		sshcmd := exec.Command("ssh", user.Username+"@"+w.workerIp, "-o", "StrictHostKeyChecking=no", "-C", cmd)
+		var sshcmd *exec.Cmd
```

Comment:
it feels like we should have a generic ssh-with-retry kind of function used by both platforms